### PR TITLE
Fix Baron2016 and Xin2016 human pancreas data sets

### PR DESCRIPTION
**Pull request by pavlin-policar**
This changes the files from `pkl.gz` to `tab.gz` to avoid concatenation issues. Also, the taxonomy id was wrong before, so I fixed that. Everything was matched through the gene name matcher as well.

Unfortunately, my laptop runs out of memory when I try to load the Baron2016 data set, so somebody should try that to make sure everything's alright.

### CODE REVIEW

File: sc/baron2016_pancreas_mouse.pkl.gz.info [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "baron2016_pancreas_mouse.pkl.gz",
+    "name": "baron2016_pancreas_mouse",
```
Comment (pavlin-policar) [inline sc/baron2016_pancreas_mouse.pkl.gz.info]: The only change to this file is that I've removed the file extension from the "name" field. @lanzagar do I also need to bump the version here?

Comment (lanzagar) [reply sc/baron2016_pancreas_mouse.pkl.gz.info]: Minor version bump sounds like the right thing.

**Comment by pavlin-policar** [comment]
@lanzagar if it's ok, please merge.

File: sc/xin2016_pancreas_human.tab.gz.info [hunk 2]
```diff
@@ -15,12 +15,12 @@
...
+    "url": "http://file.biolab.si/scdata/xin2016_pancreas_human.tab.gz",
     "references": [
         "Xin, Y, Kim, J., Okamoto, H., Ni, M., Wei, Y., Adler, C., J. Murphy, A., D. Yancopoulos, Lin, C., Gromada, J. (2016). RNA Sequencing of Single Human Islet Cells Reveals Type 2 Diabetes Genes. Cell Metabolism, 24 (4), 608-615."
     ],
     "seealso": [],
-    "taxid": "10090",
-    "num_of_genes": 39851,
-    "size": 45214358
+    "taxid": "9606",
+    "num_of_genes": 35899,
```
Comment (lanzagar) [inline sc/xin2016_pancreas_human.tab.gz.info]: Does the number of variables above need to be updated too?

Comment (pavlin-policar) [reply sc/xin2016_pancreas_human.tab.gz.info]: Yes, you're right. Nice catch.

File: sc/xin2016_pancreas_human.tab.gz.info [hunk 3]
```diff
@@ -13,14 +13,14 @@
     "year": 2016,
     "collection": "GEO",
     "instances": 1492,
-    "variables": 39851,
+    "variables": 35899,
```
Comment (lanzagar) [inline sc/xin2016_pancreas_human.tab.gz.info]: Actually it's `35900`

File: sc/baron2016_pancreas_human.tab.gz.info [hunk 4]
```diff
@@ -12,14 +12,14 @@
     "year": 2016,
     "collection": "GEO",
     "instances": 8569,
-    "variables": 20125,
+    "variables": 20053,
```
Comment (lanzagar) [inline sc/baron2016_pancreas_human.tab.gz.info]: Actually it's `20054`

**Comment by lanzagar** [comment]
I don't think we should serve datasets through Orange widgets, that crash it on most computers...
I tried the first one (baron human) and it actually crashed on my 16GB ram computer too. After closing some other programs I was able to load it, but the memory usage spiked at ~13GB while loading (after it finishes it goes down to about 6GB).

**Comment by pavlin-policar** [comment]
We agreed to remove this large data set until orange supports sparse data.

**Pull request merged**